AutoMaxLair.py

```diff
@@ -637,7 +637,7 @@ def main(log_name):
     # and the entire sequence of runs
     instance = MaxLairInstance.MaxLairInstance(
         config, com, cap, threading.Lock(), threading.Event(), log_name,
-        ENABLE_DEBUG_LOGS
+        ENABLE_DEBUG_LOGS, CHECK_ATTACK_STAT, ATTACK_STATS, CHECK_SPEED_STAT, SPEED_STATS
```

Comment:
All this new functionality is really cool. I think though that you don't need to pass these variables in individually, since `config` is already passed—you can read these values from the config object in the objects `__init__` method